### PR TITLE
Make Booster config public

### PR DESCRIPTION
**Pull request by juanjoman**
## Description
Creating a getter for the configuration to use methods like `readModelNameFromResourceName` from command handlers, event handlers, etc.

## Changes
Created a `getConfiguration` method so we can do things like this on handlers:

`const config = Booster.getConfiguration()`
`const myTableName = config.readModelNameFromResourceName('myTable')`
`sdk.dynamoDB.describeTable({ mytableName }, () => { ... })`

## Checks
- [X] Project Builds
- [X] Project passes tests and checks
- [] Updated documentation accordingly ->> Not sure if this should be documented

### CODE REVIEW

**Review by alvaroloes — COMMENTED** [review]
@juanjoman Now that I see this, I think it would be much simpler if we just make the `config` field "public" and "readonly" (it already is readonly), which exactly equivalent to what you have done, but this way it is Typescript the one which creates the getter for you :smiley:

File: packages/framework-core/src/booster.ts [hunk 1]
```diff
@@ -37,6 +37,10 @@ export class Booster {
     configurator(this.config)
   }
 
+  public static getConfiguration(): BoosterConfig {
+    return this.config
+  }
+
```
Comment (alvaroloes) [inline packages/framework-core/src/booster.ts:43]: What I meant is to remove this getter, and make the `config` getter like this:
```typescript
  *public* static readonly config = new BoosterConfig(checkAndGetCurrentEnv())
```

Comment (juanjoman) [reply packages/framework-core/src/booster.ts]: done!!

**Review by alvaroloes — APPROVED** [review]
🎉 :shipit:

**Review by charlietfe — APPROVED** [review]
:shipit:

**Review by NickSeagull — APPROVED** [review]
Thank you a lot! 🙏

**Pull request closed without merging**